.husky/pre-commit

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
```

Comment:
make sure to do git pull from `main` and `npm install`. This should not be deleted